### DISCUSSION TITLE
急需一个在vercel及netlify上部署的上传文件功能

### DISCUSSION THREAD

**Discussion by dupl** [Dogtiti/ChatGPT-Next-Web-EarlyBird#7] [General]
急需一个在vercel及netlify上部署的上传文件功能，支持PDF、word、ppt、excel、txt等格式文档…

**Comment by Dogtiti** [comment]
没太理解，vercel如何部署一个上传服务？

**Reply by alltobebetter** [reply]
我觉得可能是基于逆向api的基础上的罢

**Comment by dupl** [comment]
> 没太理解，vercel如何部署一个上传服务？

听说这个项目实现了该目的，可以在vercel上实现上传服务👉https://github.com/Hk-Gosuto/ChatGPT-Next-Web-LangChain
